builtin/game/item.lua

```diff
@@ -38,6 +38,10 @@ function core.get_pointed_thing_position(pointed_thing, above)
 end
 
 function core.dir_to_facedir(dir, is6d)
+	-- nil check for dir (for use with place_node)
+	if not dir then
+		return 0
```

Comment:
Do this: minetest.place_node(pos, {name = "default:cactus"}) and it'll error because of nil dir.